plugins/woocommerce/src/Admin/Features/ProductBlockEditor/Init.php

```diff
@@ -210,35 +218,137 @@ public function set_current_screen_to_block_editor_if_wc_admin() {
 	 * Get the product editor settings.
 	 */
 	private function get_product_editor_settings() {
+		$layout_template_registry = wc_get_container()->get( BlockTemplateRegistry::class );
+		$layout_template_logger   = BlockTemplateLogger::get_instance();
+
 		$editor_settings = array();
 
-		$template_registry     = wc_get_container()->get( BlockTemplateRegistry::class );
-		$block_template_logger = BlockTemplateLogger::get_instance();
+		foreach ( $layout_template_registry->get_all_registered() as $layout_template ) {
+			$editor_settings['layoutTemplates'][] = $layout_template->to_JSON();
+
+			$layout_template_logger->log_template_events_to_file( $layout_template->get_id() );
+			$editor_settings['layoutTemplateEvents'][] = $layout_template_logger->get_formatted_template_events( $layout_template->get_id() );
+		}
 
-		$block_template_logger->log_template_events_to_file( 'simple-product' );
-		$block_template_logger->log_template_events_to_file( 'product-variation' );
+		$product_templates = apply_filters( 'woocommerce_product_editor_get_product_templates', array() );
```

Comment:
I think a better name for this filter might be `woocommerce_product_editor_product_templates`. I'd remove the `get` from it.

Reply:
done